orte/mca/rmaps/base/rmaps_base_map_job.c

```diff
@@ -532,6 +462,7 @@ void orte_rmaps_base_map_job(int fd, short args, void *cbdata)
             }
             free(output);
         }
+        exit(1);
```

Comment:
Just want to confirm that we want to call `exit` here instead of a shutdown mechanism.